app/build.gradle

```diff
@@ -2,13 +2,15 @@ apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion 28
+    buildToolsVersion '28.0.3'
```

Comment:
Do we really need to specify build tools version here?

Reply:
It's not necessary, but it's nice to have